docs/reference/filebeat/filebeat-input-azure-blob-storage.md

```diff
@@ -336,6 +336,86 @@ filebeat.inputs:
 
 The Azure Blob Storage APIs don’t provide a direct way to filter files based on timestamp, so the input will download all the files and then filter them based on the timestamp. This can cause a bottleneck in processing if the number of files are very high. It is recommended to use this attribute only when the number of files are limited or ample resources are available.
 
+## `custom_properties` [attrib-custom-properties]
+
+Some blob properties can be `set` or `overridden` at the input level with the help of certain configuration options. Allowing users to set/override custom blob properties provides more flexibility when reading blobs from a remote storage where the user might only have read access.
+
+The currently supported `custom_properties` are :-  
```

Comment:
```suggestion
The currently supported custom properties are:
```
though I notice that this form (" :-") is used elsewhere in this input's documentation (also GCS).